apps/daffio/src/app/docs/testing/factories/doc.factory.ts

```diff
@@ -8,7 +8,96 @@ export class MockDoc implements DaffioDoc {
   title = faker.lorem.words();
   contents = faker.lorem.paragraph()
 };
-
+export const mockGuides = {
```

Comment:
It may make sense to move this to another factory?

Reply:
Addressed in 5de79c050489de6e9ea9890dcbd9152c51a22c04